Configurations/15-android.conf

```diff
@@ -1,27 +1,7 @@
 #### Android...
 #
-# It takes *one* prior-set environment variable to make it work:
-#
-# ANDROID_NDK=/some/where/android-ndk-<ver>
-#
-# As well as PATH *adjusted* to cover ${CROSS_COMPILE}gcc and company.
-#
-# Note that it's different from original instructions that required to
-# set CROSS_SYSROOT [to $ANDROID_NDK/platforms/android-<api>/arch-<arch>]
-# and CROSS_COMPILE. CROSS_SYSROOT is still recognized [and even required
-# for some legacy targets], but if not set, it's detected and set to the
-# latest Android platform available with appointed NDK automatically. If
-# you need to target older platform, pass additional -D__ANDROID_API__=N
-# to Configure. For example, to compile for ICS on ARM with NDK 10d:
-#
-# ANDROID_NDK=/some/where/android-ndk-10d
-# PATH=$ANDROID_NDK/toolchains/arm-linux-androideabi-4.8/prebuild/linux-x86_64/bin:$PATH
-# [..]./Configure android-arm -D__ANDROID_API__=14
-#
-# One can engage clang by passing CC=clang to Configure. In such case
-# PATH needs even more adjustments to cover NDK's clang itself, as well
-# as unprefixed, yet target-specific ar and ranlib [or not, if you use
-# binutils-multiarch].
+# See NOTES.ANDROID for details. [But don't miss platform-specific
+# comments below.]
```

Comment:
Like I said in the `NOTES.WIN` PR, please use round parentheses in natural languages.